frontend/src/views/LoginView.tsx

```diff
@@ -1,23 +1,45 @@
-// import UserForm from "../components/UserForm";
-// import useAuth from "../hooks/Auth";
+import UserForm from "../components/UserForm";
+import useAuth from "../hooks/Auth";
 import { useNavigate } from "react-router-dom";
 
 export default function LoginView() {
-    // const auth = useAuth();
     const navigate = useNavigate();
+    const auth = useAuth(); //Access contect using useAuth
+
+    //Signin function to handle user sign in
+    const onSubmit = async (userId: string, password: string): Promise<boolean> =>{
+        try {
+            // API call to authenticate the user
+            const response = await fetch(`/api/v1/auth/sign-in?userid=${encodeURIComponent(userId)}&password=${encodeURIComponent(password)}`, {
+                method: "POST",
+            });
+
+            //Check if signup was successful
+            if(response.ok){
+                const data = await response.json(); // Assuming the token is returned in JSON format
+                const token = { data: data.token }; // Wrap token to match Token interface
+                auth.login(token); // Save token in AuthContext               
+                //NOTE: IS THIS THE CORRECT REDIRECT PAGE???
+                navigate(`/api/v1/projects/list`) 
```

Comment:
For navigate calls, it will always be to a route registered in App.tsx. This should be "/projects"